### DISCUSSION TITLE
关于导入数据方案的问题

### DISCUSSION THREAD

**Discussion by Ethereal-sakura** [labring/FastGPT#851] [Q&A]
您好，我想请问一下FastGPT 构建知识库方案中数据导入的时候是否能通过自定义分词器来实现对文本的处理。或者要自己修改哪部分来实现这个功能？

**Comment by c121914yu** [comment]
> 您好，我想请问一下FastGPT 构建知识库方案中数据导入的时候是否能通过自定义分词器来实现对文本的处理。或者要自己修改哪部分来实现这个功能？

用自定义分割，或者 api 导入。
